prottp.go

```diff
@@ -58,24 +57,24 @@ func NewError(statusCode int, body proto.Message) ErrorWithStatus {
 	return &respError{statusCode: statusCode, body: body}
 }
 
-func Handle(mux *http.ServeMux, service Service, mws ...server.Middleware) {
+func Handle(mux *http.ServeMux, service Service, interceptor grpc.UnaryServerInterceptor, mws ...server.Middleware) {
 	sn := service.Description().ServiceName
 	fmt.Println("/" + sn)
-	hd := Wrap(service)
+	hd := Wrap(service, interceptor)
 	if len(mws) > 0 {
 		hd = server.Compose(mws...)(hd)
 	}
 	mux.Handle("/"+sn+"/", http.StripPrefix("/"+sn, hd))
 }
 
-func Wrap(service Service) http.Handler {
+func Wrap(service Service, interceptor grpc.UnaryServerInterceptor) http.Handler {
```

Comment:
Similar to `Handle`, changing the signature for `Wrap` will break any code that calls `Wrap`?

Reply:
It will break current API.

Do we need another method? like `WrapWithInterceptor`.

Reply:
> Do we need another method?

Either write another method or you need to go and fix all the projects that call it :trollface: 

(I'd probably just add `WrapWithInterceptor`...)